pkg/cli/cvd.go

```diff
@@ -56,9 +61,12 @@ func newCVDCommand(cfgFlags *configFlags) *cobra.Command {
 	create.Flags().StringVar(&createFlags.Host, hostFlag, "", "Specifies the host")
 	create.MarkFlagRequired(hostFlag)
 	create.Flags().StringVar(&createFlags.BuildID, buildIDFlag, "", "Android build identifier")
-	create.MarkFlagRequired(buildIDFlag)
 	create.Flags().StringVar(&createFlags.Target, targetFlag, "aosp_cf_x86_64_phone-userdebug",
 		"Android build target")
+	create.Flags().BoolVar(&createFlags.LocalImage, localImageFlag, false,
+		"Builds a CVD with an image files locally built, the required files are https://cs.android.com/android/platform/superproject/+/master:device/google/cuttlefish/required_images")
```

Comment:
The files listed in that page don't include `cvd-host-packages.tar.gz`.

Reply:
appended cvd-host-packages.tar.gz to the description.